### PR TITLE
use ember-cli 2.7.0 and ember-suave 4.0

### PR DESCRIPTION
**Pull request by raycohen**
versions of `ember-suave` prior to https://github.com/DockYard/ember-suave/pull/131 do not work with `ember-qunit@2.0.0`

### CODE REVIEW

**Comment by lukemelia** [comment]
Looks good beside my one question, @raycohen!

File: package.json [hunk 1]
```diff
@@ -42,8 +44,7 @@
     "button"
   ],
   "dependencies": {
-    "ember-cli-babel": "^5.1.6",
-    "ember-cli-htmlbars": "1.0.3"
```
Comment (raycohen) [inline package.json]: Fresh ember addons are generated that way. https://github.com/ember-cli/ember-addon-output/blob/master/package.json#L29

I couldn't remember if we differed for a reason

Comment (raycohen) [inline package.json]: looks like I moved it in another direction as part of the ember-cli 2.4 upgrade https://github.com/yapplabs/ember-radio-button/commit/085e4aa30d6ba7286973f357e0743d83bded0a98

Comment (raycohen) [inline package.json]: not sure why I moved it that time as the 2.4.* outputs don't show it there https://github.com/ember-cli/ember-addon-output/blob/v2.4.3/package.json#L26

**Pull request merged**